api/helpers.c

```diff
@@ -445,7 +498,7 @@ int emit_add_sub_i(mambo_context *ctx, int rd, int rn, int offset) {
       }
       return 0;
     }
-#endif
+#elif defined __aarch64__ || defined __arm__
```

Comment:
This disables the code for arm, you need to use `#endif` and `#if` instead of `#elif`